OneDrive-Cloud-Player/App.xaml.cs

```diff
@@ -1,4 +1,7 @@
 ﻿using LibVLCSharp.Shared;
+using Microsoft.AppCenter;
+using Microsoft.AppCenter.Analytics;
```

Comment:
Analytics seems to be imported but not used?

Reply:
Removed it